.git-blame-ignore-revs

```diff
@@ -0,0 +1,5 @@
+# Run this command to always ignore formatting commits in `git blame`
+# git config blame.ignoreRevsFile .git-blame-ignore-revs
+
+# Changes to be compliant with black 23.1.0
+c4c4e7b9ce37287cd515b1275558d4776bd0b75d
```

Comment:
I see `sphgeom` using this to skip a commit in `git blame` and Tim had mentioned this once for middleware packages (but I don't think we have any examples in our stack other than `sphgeom`). I'm hoping more of our packages would use this, since with the recent `black` upgrade, the files which were compliant with the old `black` formatting rules are no longer compliant. This would need introducing formatting related changes in files that the developer didn't mean to touch at all. Having this file and including the formatting related commits (and only formatting) will keep `git blame` clean.

Reply:
I had no idea about this functionality before (or if I did, I'd completely forgotten it).  Neat!

Reply:
It used to be that Github wouldn't respect this feature, but it does now!